backend/src/main/resources/templates/Ansible/instantEmailBody.html

```diff
@@ -0,0 +1,17 @@
+{#include Ansible/insightsEmailBody}
+{#content-title}Ansible report - {action.context.system_check_in.toUtcFormat()}{/content-title}
+{#content-body}
+<tr>
+    <td class="rh-content__block">
+        <p>Ansible report available for download at {action.context.system_check_in.toUtcFormat()}</p>
+    </td>
+</tr>
+<tr>
+    <td class="rh-content__block">
+        <div class="rh-stack">
+            <a class="rh-url" href="https://console.redhat.com/insights/inventory/{action.context.report_id}"><b>Download now</b></a>
```

Comment:
You will need to update the URL here.
I'm considering that you will send the report_id or something in the context. Let me know what you expect to send and I can guide you on what we need

Reply:
this link should be `https://console.redhat.com/ansible/insights/reports/{action.context.slug}`

Reply:
```suggestion
            <a class="rh-url" href="https://console.redhat.com/ansible/insights/reports/{action.context.slug}"><b>Download now</b></a>
```